<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>MainWindow</class>
	<widget class="QMainWindow" name="MainWindow">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>1000</width>
				<height>480</height>
			</rect>
		</property>
		<property name="contextMenuPolicy">
			<enum>Qt::CustomContextMenu</enum>
		</property>
		<property name="windowTitle">
			<string>MainWindow</string>
		</property>
		<widget class="QWidget" name="centralWidget">
			<layout class="QGridLayout" name="gridLayout">
				<property name="margin">
					<number>0</number>
				</property>
				<item row="0" column="0">
					<widget class="QTableView" name="tableView">
						<property name="sizePolicy">
							<sizepolicy hsizetype="Expanding" vsizetype="Expanding">
								<horstretch>0</horstretch>
								<verstretch>0</verstretch>
							</sizepolicy>
						</property>
						<property name="contextMenuPolicy">
							<enum>Qt::CustomContextMenu</enum>
						</property>
						<property name="alternatingRowColors">
							<bool>true</bool>
						</property>
						<property name="selectionBehavior">
							<enum>QAbstractItemView::SelectRows</enum>
						</property>
						<property name="showGrid">
							<bool>false</bool>
						</property>
						<property name="sortingEnabled">
							<bool>true</bool>
						</property>
						<property name="wordWrap">
							<bool>false</bool>
						</property>
						<property name="cornerButtonEnabled">
							<bool>false</bool>
						</property>
						<attribute name="horizontalHeaderCascadingSectionResizes">
							<bool>true</bool>
						</attribute>
						<attribute name="horizontalHeaderShowSortIndicator" stdset="0">
							<bool>true</bool>
						</attribute>
						<attribute name="horizontalHeaderStretchLastSection">
							<bool>true</bool>
						</attribute>
						<attribute name="verticalHeaderVisible">
							<bool>false</bool>
						</attribute>
						<attribute name="verticalHeaderVisible">
							<bool>false</bool>
						</attribute>
						<attribute name="horizontalHeaderShowSortIndicator" stdset="0">
							<bool>true</bool>
						</attribute>
						<attribute name="horizontalHeaderStretchLastSection">
							<bool>true</bool>
						</attribute>
						<attribute name="horizontalHeaderCascadingSectionResizes">
							<bool>true</bool>
						</attribute>
					</widget>
				</item>
			</layout>
		</widget>
		<!--widget class="QMenuBar" name="menuBar">
			<property name="geometry">
				<rect>
					<x>0</x>
					<y>0</y>
					<width>536</width>
					<height>23</height>
				</rect>
			</property>
			<widget class="QMenu" name="menu_File">
				<property name="title">
					<string>&amp;File</string>
				</property>
				<addaction name="action_Exit"/>
			</widget>
			<addaction name="menu_File"/>
		</widget-->
		<widget class="QToolBar" name="mainToolBar">
			<attribute name="toolBarArea">
				<enum>TopToolBarArea</enum>
			</attribute>
			<attribute name="toolBarBreak">
				<bool>false</bool>
			</attribute>
			<addaction name="actionAtHome"/>
			<addaction name="actionBackward"/>
			<addaction name="actionFordward"/>
			<addaction name="actionAtUp"/>
			<addaction name="separator"/>
		</widget>
		<widget class="QStatusBar" name="statusBar"/>
		<widget class="QDockWidget" name="dockWidget">
			<property name="sizePolicy">
				<sizepolicy hsizetype="Expanding" vsizetype="Preferred">
					<horstretch>0</horstretch>
					<verstretch>0</verstretch>
				</sizepolicy>
			</property>
			<property name="windowTitle">
				<string>Preview</string>
			</property>
			<attribute name="dockWidgetArea">
				<number>2</number>
			</attribute>
			<widget class="QWidget" name="dockWidgetContents_2">
				<layout class="QVBoxLayout" name="verticalLayout">
					<item>
						<widget class="PreviewWidget" name="previewWidget" native="true"/>
					</item>
				</layout>
			</widget>
		</widget>
		<action name="action_Exit">
		</action>
		<action name="actionFordward">
			<property name="enabled">
				<bool>false</bool>
			</property>
			<property name="icon">
				<iconset resource="rresources.qrc">
					<normaloff>:/icons/go_next.svg</normaloff>:/icons/go_next.svg</iconset>
			</property>
		</action>
		<action name="actionBackward">
			<property name="enabled">
				<bool>false</bool>
			</property>
			<property name="icon">
				<iconset resource="rresources.qrc">
					<normaloff>:/icons/go_previous.svg</normaloff>:/icons/go_previous.svg</iconset>
			</property>
		</action>
		<action name="actionAtUp">
			<property name="enabled">
				<bool>false</bool>
			</property>
			<property name="icon">
				<iconset resource="rresources.qrc">
					<normaloff>:/icons/go_up.svg</normaloff>:/icons/go_up.svg</iconset>
			</property>
		</action>
		<action name="actionAtHome">
			<property name="icon">
				<iconset resource="rresources.qrc">
					<normaloff>:/icons/gohome.svg</normaloff>:/icons/gohome.svg</iconset>
			</property>
		</action>
	</widget>
	<layoutdefault spacing="6" margin="11"/>
	<customwidgets>
		<customwidget>
			<class>PreviewWidget</class>
			<extends>QWidget</extends>
			<header>previewwidget.h</header>
			<container>1</container>
		</customwidget>
	</customwidgets>
	<resources>
		<include location="rresources.qrc"/>
	</resources>
	<connections/>
</ui>